### PR TITLE
Bulk endpoint support

### PR DESCRIPTION
**Pull request by kskumgk63**
Because of https://github.com/elimity-com/scim/issues/95, I am addressing supporting `/Bulk` endpoint handler.

- Support `/Bulk` endpoint handler
- Validate `/Bulk` request body
- Aggregate responses and errors and format them into a SCIM response.

### CODE REVIEW

**Comment by kskumgk63** [comment]
Hi, @di-wu 

I wrote a draft to implement the feature.
If there's anything odd about it, I'd appreciate it if you could point it out to me!

**Comment by di-wu** [comment]
> If there's anything odd about it, I'd appreciate it if you could point it out to me!
@kskumgk63, thank you. I'll keep an eye on it!

File: service_provider_config.go [hunk 1]
```diff
@@ -53,6 +55,43 @@ const (
...
+}
+
+// UnsupportBulkSchema specifies service provider can NOT support /Bulk endpoint.
+func UnsupportBulkSchema() BulkSchema {
+	return BulkSchema{
+		support:        false,
+		maxOperations:  defaultMaxOperations,
+		maxPayloadSize: defaultMaxPayloadSize,
+	}
+}
```
Comment (di-wu) [inline service_provider_config.go:93]: Boolean values will be false by default, no need to declare it explicitly.

For the `maxOperations` and `maxPayloadSize` I would take a look at:
https://github.com/elimity-com/scim/blob/307c22b7fcf7291d390e867f456e5b87f08f2447/service_provider_config.go#L85-L91

**Review by di-wu — COMMENTED** [review]
Keep up the good work! 🌟 
None of these comments are final, and are open for discussion.

**Comment by di-wu** [comment]
@kskumgk63 I removed `testify` as a dependency, so don't forget to rebase.

**Comment by kskumgk63** [comment]
@di-wu 
Thank you for your review!! I fixed those.

I will keep on implementing.

**Comment by di-wu** [comment]
@kskumgk63 Hey! How are you doing?

Are you still willing to take on adding support for the `/Bulk` endpoint handler?

**Comment by kskumgk63** [comment]
> @kskumgk63 Hey! How are you doing?
> 
> 
> 
> Are you still willing to take on adding support for the `/Bulk` endpoint handler?

Hey!
Yes I am.
But recently I can't have enough time to do that. 
Do you want me to implement this feature soon?

**Comment by di-wu** [comment]
No problem, take your time. I was just checking. 🙂

Let me know if you have any questions.

**Comment by di-wu** [comment]
@kskumgk63 do you plan to still work on this?

**Pull request closed without merging**